src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderLeash.java

```diff
@@ -155,6 +160,9 @@ private boolean checkLeash(Animals animal) {
     public void onInteractItem(PlayerInteractEvent event) {
         if (event.getAction() == Action.RIGHT_CLICK_BLOCK && !event.getClickedBlock().getType().isInteractable()) {
             if (capturedConf != null && capturedConf.contains("type")) {
+                if(System.currentTimeMillis() - capturedConf.getLong("capture_time") < 50) { // Less than 1 tick ago
```

Comment:
I kinda dislike this magic number, could you please make that a `private static final` constant?